src/main/java/uk/gov/hmcts/cmc/claimstore/rules/ClaimantResponseRule.java

```diff
@@ -5,9 +5,33 @@
 import uk.gov.hmcts.cmc.claimstore.exceptions.ClaimantResponseAlreadySubmittedException;
 import uk.gov.hmcts.cmc.claimstore.exceptions.ForbiddenActionException;
 import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ClaimantResponse;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ClaimantResponseType;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.CourtDetermination;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseAcceptation;
+import uk.gov.hmcts.cmc.domain.models.response.PaymentIntention;
+import uk.gov.hmcts.cmc.domain.models.response.Response;
+
+import java.util.Optional;
+
+import static uk.gov.hmcts.cmc.domain.utils.PartyUtils.isCompanyOrOrganisation;
 
 @Service
 public class ClaimantResponseRule {
+    public void isValid(Claim claim) {
+        if (!isDefendantCompanyOrOrganisation(claim)) {
+            ClaimantResponse claimantResponse = claim.getClaimantResponse().orElseThrow(IllegalArgumentException::new);
+            if (claimantResponse.getType() == ClaimantResponseType.ACCEPTATION) {
+                ResponseAcceptation responseAcceptation = (ResponseAcceptation)claimantResponse;
+                if (!Optional.ofNullable(responseAcceptation.getFormaliseOption()).isPresent()) {
```

Comment:
unnecessary wrapping of `Optional.ofNullable` here.  Don't need if block as well, you can use `orElseThrow` here